doc/manuals/user/filtering.md

```diff
@@ -53,18 +57,18 @@ result is a logic "and" between all of them.
 
 The scope correspoding to this type is "FIWARE::Filter::Existence". 
 
-      ...                                                           
-      {
-	  "restriction": {
-	      "scopes": [
-		  {
-		      "type": "FIWARE::Filter::Existence",
-		      "value": "entity::type"
-		  }
-	      ]
-	  }
-      }
-      ...                                                           
+```
```

Comment:
Please include `...` at the beginging and at the end (as in the original fragment).